.gitignore

```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
```

Comment:
I think we can simplify this git ignore. 

/node_modules and anything that you run into locally